lib/runner.js

```diff
@@ -20,6 +23,7 @@ var chainableMethods = {
 		before: {type: 'before'},
 		after: {type: 'after'},
 		skip: {skipped: true},
+		todo: {skipped: true, todo: true},
```

Comment:
I guess on the one hand todo tests should be skipped, but on the other hand they should not be counted as being skipped. I'm not even sure whether they should be counted as tests now!

Not annotating the test as `skipped` requires explicit guards in the runner where it decides whether to skip a test, but also reduces `skipped && !todo` checks where doing skip counts. Bit of a mixed bag, personally I prefer the more explicit approach.

@vdemedes @sindresorhus?